### DISCUSSION TITLE
which Pinecil firmware is this in the video?

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1372] [Q&A]
This video was posted on Reddit by someone who said that this shows different tips are not working in his pinecil.  What Firmware is this because I don't have this same Debug menu?

but the video posted on reddit shows 2 lines of debug information,  I would prefer this  Debug menu if possible since it appears to have more information?

This is not my video

https://user-images.githubusercontent.com/97197236/184793656-524e8462-8218-4dea-b5eb-196abe1d451a.mp4

**Comment by Ralim** [comment]
Thats not a debug menu, thats the detailed soldering displays.
If that user opens a discussion here I'm happy to try and figure out what is going on, it looks like something is not right.

**Reply by River-Mochi** [reply]
oh, thanks I forgot all about detailed menus since I just leave everything on default most of the time.  
they submitted  problem in Reddit Pine64  that only  Stock  Tip worked but other tips they bought did not work.
